scripts/esb_sniffer/main.py

```diff
@@ -0,0 +1,226 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+from threading import Thread, Event
+from Sniffer import Sniffer
+from time import sleep
+import logging
+import serial
+import sys
+import os
+
+PIPE_DATA = "/tmp/esb_sniffer_data"
+PIPE_COMM = "/tmp/esb_sniffer_comm"
+
+COMM_START_CAPTURE = "1"
+COMM_STOP_CAPTURE  = "2"
+
+log_lvl = logging.INFO
+
+class Process():
+    def __init__(self, name, target, daemon=True):
```

Comment:
I think we use static type declarations.
See the sbom script.
I'm not sure if this is required in NCS, but it makes code more readable and can prevent mistakes.